style.css

```diff
@@ -12,10 +14,11 @@ body {
     text-align: center;
 }
 
-h2 {
+h1 {
```

Comment:
This is more of a larger codebase CSS architecture issue, but worth discussing here:

As you needed to do with the refactor from `h2` to `h1`, by attaching your styles to a specific HTML element, this means that you're attaching semantic meaning with certain styles. When in reality, the styling is more for visual indication and less so for HTML semantics. As a result, it's often cleaner to attach these styles to a CSS class so that you are not bound by the semantic meaning of each tag (e.g., if in the future this needs to be an h3 because of how content is ordered on your page, you would need to update this too).

Let me know if you have questions about this!

Reply:
Thank you for the explanation, it's a helpful clarification on which elements to style and why! I've updated my css to reflect the style on the class rather than the h1 tag.